frontend/docs/examples.md

```diff
@@ -0,0 +1,31 @@
+---
+id: examples
+title: Examples
+hide_title: false
+hide_table_of_contents: false
+keywords:
+  - final space
+  - finalspace
+  - finalspaceapi
+  - final space wiki
+  - api
+  - restapi
+  - rest api
+  - examples
+description: Examples section for Final Space.
```

Comment:
Change this description to what this video is about.

Reply:
fixed